### PR TITLE
Create `default` test plan from `Compiler Test Plan`

### PR DESCRIPTION
**Pull request by cston**

### CODE REVIEW

**Comment by cston** [comment]
@jcouv please review

File: docs/features/default.test.md [hunk 1]
```diff
@@ -0,0 +1,173 @@
+﻿# General concerns
+- [ ] Parsing: `default < e1 > e2`
```
Comment (jcouv) [inline docs/features/default.test.md]: The regular default is `default(T)`, not `default<T>`, so I don't think there will be ambiguity.

File: docs/features/default.test.md [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+- [ ] Backward compatibility: report version error with earlier `/langversion`
```
Comment (jcouv) [inline docs/features/default.test.md]: Covered by `TestCSharp7`
`error CS8107: Feature 'default literal' is not available in C# 7. Please use language version 7.1 or greater.`

**Comment by cston** [comment]
FYI @dotnet/roslyn-compiler

**Comment by jaredpar** [comment]
Can we merge this now?

**Comment by cston** [comment]
Replaced by https://github.com/dotnet/roslyn/issues/19117.

**Pull request closed without merging**